src/OpenTracing.Contrib.NetCore/AspNetCore/MvcEventProcessor.cs

```diff
@@ -53,18 +53,18 @@ public bool ProcessEvent(string eventName, object arg)
                         // NOTE: This event is the start of the action pipeline. The action has been selected, the route
                         //       has been selected but no filters have run and model binding hasn't occured.
 
-                        var actionDescriptor = (ActionDescriptor)_beforeAction_ActionDescriptorFetcher.Fetch(arg);
+                        var actionDescriptor = GetActionDescriptor(arg);
                         var controllerActionDescriptor = actionDescriptor as ControllerActionDescriptor;
 
-                        string operationName = controllerActionDescriptor != null
-                            ? $"Action {controllerActionDescriptor.ControllerTypeInfo.FullName}/{controllerActionDescriptor.ActionName}"
-                            : $"Action {actionDescriptor.DisplayName}";
+                        string operationName = _options.ActionOperationNameResolver(actionDescriptor);
```

Comment:
any reason why this and below is not `var operationName`?

Reply:
Consistent coding style with the repo where the changes are made.